mongo/integration/sessions_test.go

```diff
@@ -330,6 +330,23 @@ func TestSessions(t *testing.T) {
 	})
 }
 
+func TestSessionsStandalone(t *testing.T) {
+	standaloneOpts := mtest.NewOptions().MinServerVersion("3.6").Topologies(mtest.Single)
+	mt := mtest.New(t, standaloneOpts)
+	mt.RunOpts("doesn't send transaction number", standaloneOpts, func(mt *mtest.T) {
```

Comment:
nit: Can we rename the test `transaction number not sent on writes` and add a comment as the first line of the test explaining what we're testing?